dashboard/test/models/sections/section_test.rb

```diff
@@ -71,6 +71,78 @@ class SectionTest < ActiveSupport::TestCase
     end
   end
 
+  test 'changing sharing_disabled to true updates user settings' do
+    student = create :student, sharing_disabled: false
+    section = Section.create @default_attrs
```

Comment:
The test mutates the created section, so using `@section` is probably inappropriate as it is shared across all tests (also the mutations would apply across the tests, being in `setup_all` rather than `setup`).